indico/modules/rb_new/util.py

```diff
@@ -30,6 +30,12 @@ def search_for_rooms(filters, only_available=False):
     if 'room_name' in filters:
         query = query.filter(Room.name.ilike('%{}%'.format(filters['room_name'])))
 
+    if 'building' in filters:
```

Comment:
you could build a dict with all the simple filters and just do `query = query.filter_by(**whatever)`